packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -20,13 +21,32 @@ export class TableRow<
     @observable
     public data?: TData;
```

Comment:
```suggestion
    public dataRecord?: TDataRecord;
```

and TDataRecord would extend TableDataRecord